src/Transition/CreateNewIDFUsingRulesV23_2_0.f90

```diff
@@ -567,6 +598,194 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
               ! If your original object starts with G, insert the rules here
 
               ! If your original object starts with H, insert the rules here
+              CASE('HVACTEMPLATE:ZONE:WATERTOAIRHEATPUMP')  ! PR 10043
+                CALL GetNewObjectDefInIDD(ObjectName,NwNumArgs,NwAorN,NwReqFld,NwObjMinFlds,NwFldNames,NwFldDefaults,NwFldUnits)
+                nodiff=.false.
+                !- remove N20 F29 Fraction of On-Cycle Power Use
+                OutArgs(1:28)=InArgs(1:28)
+                OutArgs(29:CurArgs-1)=InArgs(30:CurArgs)
+                CurArgs = CurArgs - 1
+
+              CASE('AIRLOOPHVAC:UNITARYSYSTEM')  ! PR 10043
+                  CALL GetNewObjectDefInIDD(ObjectName,NwNumArgs,NwAorN,NwReqFld,NwObjMinFlds,NwFldNames,NwFldDefaults,NwFldUnits)
+                  nodiff=.false.
+                  OutArgs(1:41)=InArgs(1:41)
+                  OutArgs(42:CurArgs-4) = InArgs(46:CurArgs)
+                  CurArgs = CurArgs - 4
+                  CALL WriteOutIDFLines(DifLfn,'AirLoopHVAC:UnitarySystem',CurArgs,OutArgs,NwFldNames,NwFldUnits)
+                  DO Num3 = 1, NumCoilLatentStuff
+                      IF (('AIRLOOPHVAC:UNITARYSYSTEM' /= CoilLatentStuff(Num3)%ParentType) .OR. (MakeUPPERCase(InArgs(1)) /= CoilLatentStuff(Num3)%ParentName)) CYCLE
+                      nodiff=.false.
+                      CALL GetNewObjectDefInIDD('Curve:Linear',NwNumArgs,NwAorN,NwReqFld,NwObjMinFlds,NwFldNames,NwFldDefaults,NwFldUnits)
+                      IF (TRIM(CoilLatentStuff(Num3)%cHeatPumpTimeConst) == '') THEN
+                          latentTau = 60
+                      ELSE
+                          READ(CoilLatentStuff(Num3)%cHeatPumpTimeConst, *) latentTau
+                      END IF
+                      IF (TRIM(CoilLatentStuff(Num3)%cMaxCyclingRate) == '') THEN
+                          latentNmax = 2.5  ! does this need to be converted to seconds?
+                      ELSE
+                          READ(CoilLatentStuff(Num3)%cMaxCyclingRate, *) latentNmax
```

Comment:
Instead of `READ`, it's somewhat safer to use 
`latentNmax = ProcessNumber(ICoilLatentStuff(Num3)%cMaxCyclingRate,ErrFlag)`

Reply:
I knew there was a better way, updating it now.